doc/user/pim.rst

```diff
@@ -434,200 +382,167 @@ cause great confusion.
    Source Group.  The keyword `fill` says to fill in all assumed data
    for test/data gathering purposes.
 
-.. index:: show ip mroute [vrf NAME] count [json]
 .. clicmd:: show ip mroute [vrf NAME] count [json]
 
    Display information about installed into the kernel S,G mroutes and in
    addition display data about packet flow for the mroutes for a specific
    vrf.
 
-.. index:: show ip mroute vrf all count [json]
 .. clicmd:: show ip mroute vrf all count [json]
 
    Display information about installed into the kernel S,G mroutes and in
    addition display data about packet flow for the mroutes for all vrfs.
 
-.. index:: show ip mroute [vrf NAME] summary [json]
 .. clicmd:: show ip mroute [vrf NAME] summary [json]
 
    Display total number of S,G mroutes and number of S,G mroutes installed
    into the kernel for a specific vrf.
 
-.. index:: show ip mroute vrf all summary [json]
 .. clicmd:: show ip mroute vrf all summary [json]
 
    Display total number of S,G mroutes and number of S,G mroutes
    installed into the kernel for all vrfs.
 
-.. index:: show ip msdp mesh-group
 .. clicmd:: show ip msdp mesh-group
 
    Display the configured mesh-groups, the local address associated with each
    mesh-group, the peer members included in each mesh-group, and their status.
 
-.. index:: show ip msdp peer
 .. clicmd:: show ip msdp peer
 
    Display information about the MSDP peers. That includes the peer address,
    the local address used to establish the connection to the peer, the
    connection status, and the number of active sources.
 
-.. index:: show ip pim assert
 .. clicmd:: show ip pim assert
 
    Display information about asserts in the PIM system for S,G mroutes.
 
-.. index:: show ip pim assert-internal
 .. clicmd:: show ip pim assert-internal
 
    Display internal assert state for S,G mroutes
 
-.. index:: show ip pim assert-metric
 .. clicmd:: show ip pim assert-metric
 
    Display metric information about assert state for S,G mroutes
 
-.. index:: show ip pim assert-winner-metric
 .. clicmd:: show ip pim assert-winner-metric
 
    Display winner metric for assert state for S,G mroutes
 
-.. index:: show ip pim group-type
 .. clicmd:: show ip pim group-type
 
    Display SSM group ranges.
 
-.. index:: show ip pim interface
 .. clicmd:: show ip pim interface
 
    Display information about interfaces PIM is using.
 
-.. index:: show ip pim mlag [vrf NAME] interface [detail|WORD] [json]
 .. clicmd:: show ip pim mlag [vrf NAME|all] interface [detail|WORD] [json]
 
    Display mlag interface information.
 
-.. index:: show ip pim [vrf NAME] join [A.B.C.D [A.B.C.D]] [json]
```

Comment:
losing the various flags to the show command?

Reply:
Should have been in the clicmd definition